fastestimator/util/data.py

```diff
@@ -28,4 +28,4 @@ def write_without_log(self, key: str, value: Any):
         self.maps[1][key] = value
 
     def read_logs(self, extra_keys: Set[str]) -> Dict[str, Any]:
-        return {**{k: v for k, v in self.maps[1].items() if k in extra_keys}, **self.maps[0]}
+        return {**{k: self.__getitem__(k) for k in extra_keys if self.__contains__(k)}, **self.maps[0]}
```

Comment:
this will actually be slower than the other implementation with no functional benefits

Reply:
right, just realized, changed back.